README.md

```diff
@@ -57,6 +57,27 @@ This will start the development server on your local machine and allow you to st
    ```
    npm run new
    ```
+4. Run linter by the following command:
+   
+   ```
+   npm run lint
+   ```
+
+5. Run all test using the following command.
+
+   ```
+   npm run test
+   ``` 
+> Note: Since packages have interdependency, you need to run `dev` script atleast once before running `lint` and `test` scripts globally.
+
+<a id="point-6"></a>
+
+>Above commands are global commands, they run for the entire monorepo which includes all the packages, apps and docs etc but these commands can be coupled with `-w` or `-workspace` flag to run them on a specific repo/workspace inside this monorepo.
+1. To run test script on the package `elixir-ui`, use the following command:
```

Comment:
Point 6.

Reply:
Change this and please merge @JaeAeich!

Reply:
can you please elaborate on what to change @git-anurag-hub, do I have to remove point 6?